### PR TITLE
VidCloud9 Update with Settings

### PR DESCRIPTION
**Pull request by Gurrman375**
VidCloud9 update with Settings
image
image
image
image

Settings:

No Elapsed Time:
image

No Video Time:
image

No Info:
image

### CODE REVIEW

File: VidCloud9/dist/metadata.json [hunk 1]
```diff
@@ -1,18 +1,38 @@
...
-  "iframe": true
-}
+  {
+    "author": {
+      "name": "GURRMAN375D",
+      "id": "446528923535278082"
+    },
+    "service": "VidCloud9",
+    "description": {
+      "en": "VidCloud9 is a great place to find shows you missed or movies your dieing to see."
```
Comment (Alanexei) [inline VidCloud9/dist/metadata.json:8]: Rename `dieing` to `dying`

Comment (Gurrman375) [reply VidCloud9/dist/metadata.json]: Fixed

File: VidCloud9/dist/metadata.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     },
     "service": "VidCloud9",
     "description": {
-      "en": "VidCloud9 is a great place to find shows you missed or movies your dieing to see."
+      "en": "VidCloud9 is a great place to find shows you missed or movies your dying to see."
```
Comment (Cairo2k18) [inline VidCloud9/dist/metadata.json:8]: You should change `your` to `you're`

File: VidCloud9/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
```
Comment (Bas950) [inline VidCloud9/tsconfig.json:6]: https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

Comment (Gurrman375) [reply VidCloud9/tsconfig.json]: fixed

**Pull request merged**